### PR TITLE
v7-docs

### PR DESCRIPTION
**Pull request by tobiasnteireho**
update readme for v7. Includes build instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +76,7 @@ The file viewer.html contains several endpoints to allow data to be be exported
...
 
-`setDataURL` - **\[optional]** location of where you want to send the data from the interview when the use SENDs or SUBMITs the completed interview. This must be some website-based program that can "catch" the XML data stream that contains the user's answers to your interview in HotDocs .anx format.
+`setDataURL` - **\[optional]** location of where you want to send the data from the interview when the use SENDs or SUBMITs the completed interview. This must be some website-based program that can "catch" the XML data stream that contains the user's answers to your interview in HotDocs .anx format and in an equivalent json format through an http POST.
```
Comment (mitch-mitchel) [inline README.md]: user or end user

File: README.md [hunk 2]
```diff
@@ -4,29 +4,68 @@ This repo hosts the distributable production version of the A2J Viewer.
...
+## Installing from zip
+Download the latest viewer from the releases page https://github.com/CCALI/a2jviewer/releases. Unzip the viewer package into your webroot or preferred directory on your web server. The Apache user, for example `www-data` or `apache`, should own the folder where you place the viewer. Settings for configuring the viewer are found in the `viewer.html` file. The default settings work with the Demo Widget (see below), but should be updated to match your `guides` directory on your server. For example, you will most likely want to store A2J Guided Interview® files in a directory that is populated internally or with your own file upload interface.
+
+##Installing from source
+
+### Install build tools
+
+1.)  Install nvm
+The DAT is a simple restful API that requires nodejs to serve endpoints. Though, you are free to install the node version that the DAT targets and manage it manually, the recommended method is to use a node version manager which will allow the simultaneous installation of multiple versions of node and mitigates several administration issues.
```
Comment (mitch-mitchel) [inline README.md]: should this be viewer related instruction? if it is meant to be the DAT install instructions, should it be marked as optional?

File: README.md [hunk 3]
```diff
@@ -4,29 +4,68 @@ This repo hosts the distributable production version of the A2J Viewer.
...
+OR the equivalent
+
+```
+npm install
+npm run build
+mv index.html index.dev.html
+mv index.production.html index.html
+```
 
 ## Upgrading
```
Comment (mitch-mitchel) [inline README.md:69]: maybe make special mention of the special endpoints here?

**Review by mitch-mitchel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tobiasnteireho) [reply README.md]: it's to allow people to build from the repo. Fixed in latest commit.

File: README.md (hunk 1 above)
Comment (tobiasnteireho) [reply README.md]: I don't understand this comment.

Comment (mitch-mitchel) [reply README.md]: highlighted the wrong stuff `.... when the use SENDs or SUBMITs`  seems like `use` should be `user`

**Review by JessicaFrank — APPROVED** [review]
Looks good to me!

**Review by JessicaFrank — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (tobiasnteireho) [reply README.md:69]: added note in latest commit

File: README.md (hunk 1 above)
Comment (tobiasnteireho) [reply README.md]: fixed in later commit

**Review by mitch-mitchel — APPROVED** [review]

**Pull request merged**